pytest.ini

```diff
@@ -27,6 +27,7 @@ markers =
     # Cluster markers
     ## Architecture support
     arm64: Tests that can run on ARM-based cluster
+    skip_s390x: Tests that are disabled on s390x-based cluster
```

Comment:
instead of skipping - all s390x tests should be marked explicitly, same as arm64

Reply:
Just to confirm my understanding: for the compute tests, we need to disable around 10 tests based on my estimation. Does that mean we should explicitly mark the remaining ~300 tests that should not be skipped as s390x?